src/storage/FileResourceStore.ts

```diff
@@ -3,7 +3,7 @@ import { createReadStream, createWriteStream, promises as fsPromises } from 'fs'
 import { posix } from 'path';
 import type { Readable } from 'stream';
 import { DataFactory } from 'n3';
-import type { Quad } from 'rdf-js';
+import type { NamedNode, Quad } from 'rdf-js';
```

Comment:
Why does this class still exist if we have the accessor? Performance?

Reply:
I refer to my comment below